samples/dnn/text_detection.cpp

```diff
@@ -2,22 +2,14 @@
     Text detection model: https://github.com/argman/EAST
     Download link: https://www.dropbox.com/s/r2ingd0l3zt8hxs/frozen_east_text_detection.tar.gz?dl=1
 
-    CRNN Text recognition model taken from here: https://github.com/meijieru/crnn.pytorch
-    How to convert from pb to onnx:
-    Using classes from here: https://github.com/meijieru/crnn.pytorch/blob/master/models/crnn.py
```

Comment:
Why is this information dropped?

Reply:
The previous models are taken from this link, but I also provide some other models.
How about put this information into the tutorial in detail? or just leave it here?
Do you have any suggestions?

Reply:
I have added the necessary information to my tutorial.

Reply:
I have put these information back to it.